### PR TITLE
Workaround for #62 (attempt 2)

### PR DESCRIPTION
**Pull request by anbrsap**
Use a fork of [jenkinsfile-runner](https://github.com/jenkinsci/jenkinsfile-runner) that contains a patch working around #62.

The fork is located in this repository in branches and tags starting with `jenkinsfile-runner--`.

### CODE REVIEW

File: jenkinsfile-runner-base-image/packager-config.yml [hunk 1]
```diff
@@ -15,8 +15,11 @@ buildSettings:
         #TODO: replace once Jenkinsfile Runner archives runner & CWP is updated
         noCache: true
       source:
-        git: https://github.com/jenkinsci/jenkinsfile-runner.git
-        branch: "1.0-beta-19"
+        # Use fork until fixes are available in upstream
+        # git: https://github.com/jenkinsci/jenkinsfile-runner.git
+        # branch: "1.0-beta-19"
+        git: https://github.com/SAP/stewardci-jenkinsfilerunner-image
+        branch: "jenkinsfile-runner--1.0-beta-19-steward"
```
Comment (anbrsap) [inline jenkinsfile-runner-base-image/packager-config.yml:22]: This must be changed to a tag once #67 has been merged and tagged.

File: jenkinsfile-runner-base-image/packager-config.yml [hunk 2]
```diff
@@ -15,8 +15,11 @@ buildSettings:
         #TODO: replace once Jenkinsfile Runner archives runner & CWP is updated
         noCache: true
       source:
-        git: https://github.com/jenkinsci/jenkinsfile-runner.git
-        branch: "1.0-beta-19"
+        # Use fork until fixes are available in upstream
+        # git: https://github.com/jenkinsci/jenkinsfile-runner.git
+        # branch: "1.0-beta-19"
+        git: https://github.com/SAP/stewardci-jenkinsfilerunner-image
+        branch: "jenkinsfile-runner--1.0-beta-19-steward-1"
```
Comment (rinckm) [inline jenkinsfile-runner-base-image/packager-config.yml:22]: As far as I can see this branch has not the correct content.
Only branch jenkinsfile-runner--1.0-beta-19-steward has similar content as 
https://github.com/jenkinsci/jenkinsfile-runner.git

Comment (anbrsap) [reply jenkinsfile-runner-base-image/packager-config.yml:22]: The Git repo is also wrong :-(

Comment (anbrsap) [reply jenkinsfile-runner-base-image/packager-config.yml:22]: Actually there should be only a tag `jenkinsfile-runner--1.0-beta-19-steward`.

But I don't know how a branch with the same name but different content has been created. I've deleted the branch.

**Review by rinckm — APPROVED** [review]

**Comment by anbrsap** [comment]
Successfully tested with image `stewardci/stewardci-jenkinsfile-runner:TEST_210127_0114bbd`.

**Review by alxsap — APPROVED** [review]

**Pull request merged**